com.reprezen.swagedit/src/com/reprezen/swagedit/editor/DocumentUtils.java

```diff
@@ -74,14 +74,7 @@ public static IFile getWorkspaceFile(URI uri) {
 		IWorkspaceRoot root = ResourcesPlugin
 				.getWorkspace()
 				.getRoot();
-
-		URI relativize = root.getLocationURI().relativize(uri);
-		IPath path = new Path(relativize.getPath());
-		try {
-			return root.getFile(path);
-		} catch (Exception e) {
-			return null;
-		}
+		return root.getFileForLocation(new Path(uri.getPath()));
 	}
```

Comment:
Any idea why the previous code was catching `exception`?  Is there a need to do something similar here, or does getFileForLocation manage the expected exception (whatever exception might have been anticipated in the previous code)?